can/interfaces/__init__.py

```diff
@@ -27,4 +32,11 @@
     "neousys": ("can.interfaces.neousys", "NeousysBus"),
 }
 
+BACKENDS.update(
+    {
+        interface.name: (interface.module_name, interface.attrs[0])
```

Comment:
I think importlib uses slightly different attributes. You could test it by installing python-can-remote (on both Python 3.9 and older):

```
pip install python-can-remote
```

Reply:
That helped very much, thank you. It turns out I was misreading the documentation for importlib. I have gone ahead and added a fix that properly utilizes importlib now.